pkg/controllers/vdb/podfacts.go

```diff
@@ -1290,3 +1290,11 @@ func (p *PodFacts) quorumCheckForRestartCluster(restartOnly bool) bool {
 	})
 	return restartablePrimaryNodeCount >= (primaryNodeCount+1)/2
 }
+
+// IfNoPodsInSandbox returns true if we cannot find any pods in the target sandbox
+func (p *PodFacts) IfNoPodsInSandbox(sandbox string) bool {
```

Comment:
```suggestion
func (p *PodFacts) AnyPodsInSandbox(sandbox string) bool {
```
Or `IsSandboxEmpty()`